docs/user-guide/researcher/tensorboard.md

```diff
@@ -33,6 +33,9 @@ exp = Experiment(tags=tags,
                 )
 ```
 
+!!! info "Validation facility"
+    Tensorboard displays results got from validation metric, in the testing facility. For each `Round`, values will be exported into `Tensorboard`. If `test_batch_size` is set to a specific value, each computed value will be repported (depending on the size of `test_batch_size` set).
```

Comment:
```suggestion
    Tensorboard displays the results of the validation/testing. During each `Round`, values will be exported into `Tensorboard` in real time. If `test_batch_size` is set to a specific value, each computed value will be reported (depending on the size of `test_batch_size` set).
```

Reply:
I would update your suggestion that way: 

Tensorboard displays the results of the validation/testing steps. During each `Round`, values will be exported into `Tensorboard` in real time. If `test_batch_size` is set to a specific value, each computed value will be reported (depending on the size of `test_batch_size` set).